src/mantine-dates/src/components/TimeInputBase/AmPmInput/AmPmInput.tsx

```diff
@@ -34,16 +34,16 @@ export const AmPmInput = forwardRef<HTMLInputElement, AmPmSelectProps>(
     };
 
     const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
-      if (event.nativeEvent.code === 'ArrowUp' || event.nativeEvent.code === 'ArrowDown') {
+      if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
```

Comment:
I think we should leave `event.nativeEvent.code` for arrows since they are universal for all languages and code is more safe

Reply:
Okay, I have undone the change